app/queries/claims/mentors_with_remaining_claimable_hours_query.rb

```diff
@@ -9,9 +9,13 @@ def call
 
   def remaining_claimable_hours_condition(scope)
     mentor_with_claimable_hours = scope.filter do |mentor|
-      Claims::Mentor::CalculateRemainingMentorTrainingHoursForProvider.call(mentor:, provider: params[:provider], claim: params[:claim]).positive?
+      training_allowance_for(mentor).available?
     end
 
     scope.where(id: mentor_with_claimable_hours)
   end
+
+  def training_allowance_for(mentor)
+    Claims::TrainingAllowance.new(mentor:, provider: params[:provider], academic_year: Claims::ClaimWindow.current.academic_year)
+  end
```

Comment:
I think this also needs to follow the same pattern as changed above.

`claim: params[:claim]` in this case.

Reply:
Resolved in 27493236d162f01f71c911d458302096f86fc0ee